src/accounts/single_account_request.rs

```diff
@@ -1,26 +1,50 @@
 use crate::models::{is_public_key, Request};
 
-/// SingleAccountRequest is the request for the /accounts enpoint to get a single account.
-/// [More Details](https://www.stellar.org/developers/horizon/reference/endpoints/accounts.html "Accounts")
-pub struct SingleAccountRequest {
+#[derive(Default, Clone)]
+pub struct AccountId(String);
+#[derive(Default, Clone)]
+pub struct NoAccountId;
```

Comment:
We should add some documentation on the context of each of these states. What do they mean,?